Formula/chromedriver.rb

```diff
@@ -1,16 +1,18 @@
 class Chromedriver < Formula
   desc "Tool for automated testing of webapps across many browsers"
   homepage "https://sites.google.com/a/chromium.org/chromedriver/"
-  url "https://chromedriver.storage.googleapis.com/2.21/chromedriver_mac32.zip"
-  version "2.21"
-  sha256 "41f21005b498adc5896e4c3e264d718e4ade09609a7544ea8753d4e8ef1e787f"
+  url "https://chromedriver.storage.googleapis.com/2.22/chromedriver_mac32.zip"
+  version "2.22"
+  sha256 "086943786ce736c72f790d85bc810cd541d97abdf81d51e30da62a6d3e2152e3"
 
   bottle :unneeded
 
   def install
     bin.install "chromedriver"
   end
 
+  plist_options :startup => "true", :manual => "chromedriver"
```

Comment:
@MikeMcQuaid My fault!  I started off with just `:manual` in 7dd393f0b6d969ca9881e6a777bf7f0b76311479, but after looking at the pre-existing `plist` def and its `RunAtLoad` `<true/>`, I added the `:startup` to try and match.  Is the launchagent per-user but the plist_options a boot setting then?

http://www.rubydoc.info/github/Homebrew/brew/Formula.plist_options suggested that having both gave the user a choice, so I didn't think it was harmful... (but I am aware that's a third-party site - I struggled to find what the audit failure meant when I was just trying to update the version :( )